README.md

```diff
@@ -140,7 +140,7 @@ Breaking down that command:
 
 ## Payments
 
-The connector will facilitate an interledger payment upon receiving a notification for a transfer in which it is credited. That "source" transfer must have a `ilp_header` in its credit's `memo` that specifies the payment's destination and amount.
+The connector will facilitate an interledger payment upon receiving a notification for a transfer in which it is credited. That "source" transfer must have an `IlpHeader` in its credit's `memo.ilp` that specifies the payment's destination and amount.
```

Comment:
Should be ILP Packet or ILP Payment Packet instead of ILP Header (we should make the naming consistent throughout, there are a couple more mentions of "header" in this PR too)